packages/database/src/migrations/20210816012839-AddProjectPacmossi-modifies-data.js

```diff
@@ -0,0 +1,141 @@
+};
+
+const countryCodes = ['FJ', 'KI', 'NR', 'NU', 'PG', 'SB', 'TV', 'TO', 'VU'];
+
+const projectCode = 'pacmossi';
+const projectName = 'PacMOSSI';
+const projectDescription = 'Pacific Mosquito Surveillance Strengthening for Impact';
+
+const permissionGroups = ['PacMOSSI', 'PacMOSSI Senior'];
+
+const dashboardName = 'Vector Surveillance';
+
+const projectDashboardItemCode = 'project_details';
+const projectDashboardCode = `${projectCode}_${projectName}`;
+
+const getCountryDashboardCode = countryCode => `${countryCode}_${projectName}`;
+
+const hierarchyNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM entity_hierarchy WHERE name = '${name}'`);
+  return record.rows[0] && record.rows[0].id;
+};
+
+const addCountryToProject = async (db, countryCode) => {
+  await insertObject(db, 'entity_relation', {
+    id: generateId(),
+    parent_id: await codeToId(db, 'entity', projectCode),
+    child_id: await codeToId(db, 'entity', countryCode),
+    entity_hierarchy_id: await hierarchyNameToId(db, projectCode),
+  });
+  await insertObject(db, 'dashboard', {
+    id: generateId(),
+    code: getCountryDashboardCode(countryCode),
+    name: dashboardName,
+    root_entity_code: countryCode,
+    sort_order: 1,
+  });
+};
+
+const addItemToDashboard = async (
+  db,
+  { code, dashboardCode, permissionGroup, entityTypes, projectCodes },
+) => {
+  const dashboardItemId = (await findSingleRecord(db, 'dashboard_item', { code })).id;
+  const dashboardId = (await findSingleRecord(db, 'dashboard', { code: dashboardCode })).id;
+  await insertObject(db, 'dashboard_relation', {
+    id: generateId(),
+    dashboard_id: dashboardId,
+    child_id: dashboardItemId,
+    entity_types: `{${entityTypes.join(', ')}}`,
+    project_codes: `{${projectCodes.join(', ')}}`,
+    permission_groups: `{${permissionGroup}}`,
+    sort_order: 1,
+  });
+};
+
+exports.up = async function (db) {
+  await insertObject(db, 'entity', {
+    id: generateId(),
+    code: projectCode,
+    parent_id: await codeToId(db, 'entity', 'World'),
+    name: projectName,
+    type: 'project',
+  });
+
+  await insertObject(db, 'entity_hierarchy', {
+    id: generateId(),
+    name: projectCode,
+  });
+
+  await Promise.all(countryCodes.map(countryCode => addCountryToProject(db, countryCode)));
+
+  // insert project dashboard
+  await insertObject(db, 'dashboard', {
+    id: generateId(),
+    code: projectDashboardCode,
+    name: dashboardName,
+    root_entity_code: projectCode,
+    sort_order: 1,
+  });
+
+  // add project details dashboard item and relations
+  await addItemToDashboard(db, {
+    code: projectDashboardItemCode,
+    dashboardCode: projectDashboardCode,
+    permissionGroup: `${permissionGroups[0]}`,
+    entityTypes: ['project'],
+    projectCodes: [`${projectCode}`],
+  });
+
+  await insertObject(db, 'project', {
+    id: generateId(),
+    code: projectCode,
+    description: projectDescription,
+    sort_order: 16,
```

Comment:
```suggestion
    sort_order: 15,
```

Got ya!

Reply:
Ah it must start at 0, thank you @billli0 !